Components/Modal/Modal.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./modalstyles.css" />
+    <title>Modal Component</title>
+  </head>
+  <body>
+    <div>
+      <h1>Simple Modal</h1>
+      <div class="modal">
+        <h2 class="modal-title">
+          Modal Title<button class="close-modal">x</button>
+        </h2>
+
+        <hr />
+        <p class="modal-txt">This is a simple modal</p>
+        <hr />
+        <div class="modal-btns">
+          <button class="modal-btn">Close</button>
```

Comment:
reuse your button component and style modal button accordingly
```suggestion
          <button class="btn btn-modal">Close</button>
```